### PR TITLE
[ADD] partner_contact_creation_security: new module

### PR DESCRIPTION
**Pull request by merchantroad**
Compare my custom module to limit users to only creating/editing child contacts of  parent contacts.

### CODE REVIEW

**Comment by dreispt** [comment]
Thank you for contributing @merchantroad .
I understand this is a WIP and your are cleaning up the code.
Here is the relevant part of the current Travis log: https://travis-ci.com/github/OCA/partner-contact/jobs/432792320#L342

File: partner_contact_creation_security/security/res_partner_security.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		<!--restrict read access-->
+		<field name="perm_read" eval="False"/>
+		<!--give edit/update access-->
+		<field name="perm_write" eval="True"/>
+		<!--give create access-->
+		<field name="perm_create" eval="True"/>
+		<!--restrict delete access-->
+		<field name="perm_unlink" eval="False"/>
+	</record>
+</odoo>
```
Comment (emagdalenaC2i) [inline partner_contact_creation_security/security/res_partner_security.xml:31]: Delete extra character

**Comment by merchantroad** [comment]
Good Morning Eduardo,

Thank you for reviewing my code.  I'm not exactly sure what you mean by "Delete extra character" in this file.  Please clarify what character you are referring to.

Thank
________________________________

**Comment by emagdalenaC2i** [comment]
@merchantroad sorry, it was a "No newline at end of file" message
image

**Comment by NL66278** [comment]
You can add the newline character to the xml using a good editor (vim !), by adding an empty line and then removing it again, then save the file. Sounds strange, but works.

**Comment by NL66278** [comment]
For the rest: use tools like pylint and black to format and check your code for conventions, and then push again, to have proper travis tests.

**Pull request closed without merging**